### PR TITLE
Feature/ci

### PR DESCRIPTION
**Pull request by greymd**
Changes

* Add some unit tests
* CI settings for travic-ci (travis-ci.org/greymd/wpbcopy)
  + Automatically unit tests start in case of pushing something or creating P-R.
* Fix small bug (138f2c0)

### CODE REVIEW

File: test/test.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    WPB_ID="$(cat /dev/urandom | strings | grep -o '[[:alnum:]]' | tr -d '\n' | fold -w 128 | head -n 1)"
+    WPB_PASSWORD="$(cat /dev/urandom | strings | grep -o '[[:alnum:]]' | tr -d '\n' | fold -w 128 | head -n 1)"
+}
+
+test_simple_string () {
+    echo "aaaa" | tr -d '\n' | wpbcopy
+    assertEquals 0 $?
+    assertEquals "aaaa" "$(wpbpaste)"
+}
+
```
Comment (mecab) [inline test/test.sh:30]: How about inserting `test_include_whitespaces`, i.e. `"aaa bbb ccc"`

Comment (greymd) [reply test/test.sh:30]: done 🍛

Comment (mecab) [reply test/test.sh:30]: LGTM🎉

**Comment by mecab** [comment]
LGTM🎉

Let's merge them.

2017年1月18日 18:01 +0900 に Yasuhiro, Yamada <notifications@github.com> は書きました:

**Pull request merged**